Support custom types in Action Server UI

rihardsgravis opened the pull request:
Support parsing Action's Custom Types in Action Server UI run action view.


Closes #292

rihardsgravis committed:
Support custom types in Action Server UI

rihardsgravis committed:
Fix tests

fabioz reviewed (approved):
Nice touch with adding the tests ;)

I think that the only thing missing is an entry to the CHANGELOG.md.

rihardsgravis committed:
Update to use OpenAPI types

rihardsgravis committed:
Add Changelog entry

rihardsgravis commented:
@fabioz I also removed some of the custom defined types and added the official OpenAPI schema types dependency. 
Some of the more advanced type definitions _may_ break - please let me know if you encounter a case
